etcd/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/hertz-contrib/registry/etcd
 go 1.16
 
 require (
+	github.com/bytedance/sonic v1.8.3 // indirect
```

Comment:
This doesn't look like an indirect dependency.
> go mod tidy

The version here is not the same as `eureka`? can we both use **v1.8.3**?